src/helpers/axios-interceptors.ts

```diff
@@ -22,12 +25,37 @@ export const setupAxios = () => {
       return response
     },
     (error) => {
-      Logger(ConsoleLogType.ERROR, {
-        service: 'API Rest',
-        message: `HTTP ${error?.response?.status} - ${JSON.stringify(error?.response?.data)}`,
-        error
-      })
+      if (error?.name === 'CanceledError') {
+        Logger(ConsoleLogType.ERROR, {
+          service: 'API Rest',
+          message: `Axios Cancel: ${error?.message}`,
+          error
+        })
+      } else {
+        Logger(ConsoleLogType.ERROR, {
+          service: 'API Rest',
+          message: `HTTP ${error?.response?.status} - ${JSON.stringify(error?.response?.data)}`,
+          error
+        })
+      }
+
       return Promise.reject(error)
     }
   )
 }
+
+export const includeAxiosAuthHeaders = () => {
+  axios.interceptors.request.use(async (config) => {
+    if (!(config.url?.includes('auth') && config.method === 'post')) {
+      await getToken().then((storageToken) => {
```

Comment:
If you use await you can avoid using then

```suggestion
      const storageToken = await getToken()
```